source/docs/software/advanced-controls/trajectories/holonomic.rst

```diff
@@ -0,0 +1,86 @@
+Holonomic Drive Controller
+==========================
+The holonomic drive controller is a trajectory tracker for robots with holonomic drivetrains (i.e. swerve and mecanum). This can be used to accurately track trajectories with correction for minor disturbances.
```

Comment:
```suggestion
The holonomic drive controller is a trajectory tracker for robots with holonomic drivetrains (i.g. swerve, mecanum, etc.). This can be used to accurately track trajectories with correction for minor disturbances.
```

Reply:
You mean `e.g.`?

Reply:
I do in fact mean e.g. I've updated the request